pkg/signature/keys.go

```diff
@@ -43,6 +43,25 @@ import (
 	_ "github.com/sigstore/sigstore/pkg/signature/kms/hashivault"
 )
 
+var (
+	// Fulcio cert-extensions, documented here: https://github.com/sigstore/fulcio/blob/main/docs/oid-info.md
+	CertExtensionIssuer                   = "1.3.6.1.4.1.57264.1.1"
```

Comment:
nit: Can we name this `CertExtensionOIDCIssuer` to distinguish this extension from the issuer of a certificate (e.g. fulcio)?